src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTabId = tabs.some(item => item.id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+  const getSelected = () => tabs.find(el => el.id === currentTabId);
```

Comment:
you can do just method find and if element is not found, use first tab: `tabs.find(el => el.id === currentTabId) || tabs[0]`